lib/src/clojure_lsp/feature/completion.clj

```diff
@@ -217,36 +217,38 @@
                                 set)]
       (concat
         (when (simple-ident? cursor-value)
-          (->> aliases
-               (filterv (fn [element]
-                          (or
-                            (matches-fn (:alias element))
-                            (matches-fn (:to element)))))
-               (mapv
-                 (fn [element]
-                   (let [require-edit (some-> cursor-loc
-                                              (f.add-missing-libspec/add-known-alias (symbol (str (:alias element)))
-                                                                                     (symbol (str (:to element)))
-                                                                                     db)
-                                              r.transform/result)]
-                     (cond-> (element->completion-item element nil :required-alias)
-                       (seq require-edit) (assoc :additional-text-edits (mapv #(update % :range shared/->range) require-edit))))))))
-        (->> (:analysis db)
-             (mapcat val)
-             (keep
-               #(when (and (identical? :var-definitions (:bucket %))
-                           (not (:private %))
-                           (contains? alias-namespaces (:ns %))
-                           (or (simple-ident? cursor-value) (matches-fn (:name %))))
-                  [(:ns %) (element->completion-item % cursor-alias :unrequired-alias)]))
-             set
-             (map
-               (fn [[element-ns completion-item]]
-                 (let [require-edit (some-> cursor-loc
-                                            (f.add-missing-libspec/add-known-alias (symbol cursor-alias) element-ns db)
-                                            r.transform/result)]
-                   (cond-> completion-item
-                     (seq require-edit) (assoc :additional-text-edits (mapv #(update % :range shared/->range) require-edit)))))))))))
+          (into []
```

Comment:
Took the oportunity to transform this code to a transducer, it makes easier to review this piece with Github "hide whitespace" option enabled:
![image](https://user-images.githubusercontent.com/7820865/161890901-ffc64383-1ae0-453f-9882-0a50bba29ee7.png)